frontend/src/metabase/query_builder/components/QueryModals.tsx

```diff
@@ -345,16 +342,6 @@ class QueryModals extends Component<QueryModalsProps> {
             <PreviewQueryModal onClose={onCloseModal} />
           </Modal>
         );
-      case MODAL_TYPES.CONVERT_QUERY:
```

Comment:
I thought that we wanted to preserve the modal for small screens, see [Slack](https://metaboat.slack.com/archives/C01MS7DQKR6/p1709893938136189?thread_ts=1709748018.731169&cid=C01MS7DQKR6) & [Figma](https://www.figma.com/file/TZseFDIOAhNU6ArKtFmxBx/Show-generated-SQL-in-a-side-bar-automatically?type=design&node-id=86-6055&mode=design&t=2mKsvplNszKwQLBz-4)

Reply:
Those sections are outdated.
The sidebar will take the full width of the notebook on small screens so you will either see the notebook or the SQL preview. It achieves the same thing as the modal but without having to switch components.